code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -198,7 +202,12 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 /datum/game_mode/dynamic/send_intercept()
 	. = "<b><i>Central Command Status Summary</i></b><hr>"
-	switch(round(threat_level))
+	var/shown_threat
+	if(prob(FAKE_REPORT_CHANCE))
+		shown_threat = rand(1 to 100)
```

Comment:
tbh, I think that this bit isn't needed. just increase the divergence in shown threat that you apply below to something like +/- 25 instead of +/- 15.

Reply:
no